recipes/pcl/all/conanfile.py

```diff
@@ -86,6 +86,8 @@ class PclConan(ConanFile):
         # "with_rssdk": [True, False],
         # "with_rssdk2": [True, False],
         # "with_qvtk": [True, False],
+        # If to enable https://github.com/PointCloudLibrary/pcl/pull/5764 
+        "with_compilation_optimisation": [True, False],
```

Comment:
```suggestion
        # Enables https://github.com/PointCloudLibrary/pcl/pull/5764
        "instantiate_only_core_point_types": [True, False],
```
The option name could be made more specific, IMO. It's a bit vague otherwise.